nexus-repository-conan/src/main/java/org/sonatype/repository/conan/internal/hosted/v1/ConanHostedFacet.java

```diff
@@ -145,6 +146,15 @@ public String getDownloadUrlAsJson(final ConanCoords coords) throws JsonProcessi
     return generateDownloadPackagesUrlsAsJson(coords, repositoryUrl);
   }
 
+  public String getDigestAsJson(final ConanCoords coords) throws JsonProcessingException {
+    String repositoryUrl = getRepository().getUrl();
+    Map<String, String> digest = new HashMap<>();
+    digest.put(AssetKind.CONAN_MANIFEST.getFilename(),
+        repositoryUrl + "/" + ConanHostedHelper.getHostedAssetPath(coords, AssetKind.CONAN_MANIFEST));
+
+    return ConanHostedHelper.MAPPER.writeValueAsString(digest);
+  }
+
```

Comment:
Please move this method to ConanHostedMetadataFacetSupport

Reply:
Done 8ad909ca4ee24edbaea6287a237d5d9f4f7e6220